server/pulp/server/managers/consumer/group/cud.py

```diff
@@ -78,9 +66,7 @@ def create_consumer_group(group_id, display_name=None, description=None, consume
         try:
             collection.insert(consumer_group, safe=True)
         except DuplicateKeyError:
-            raise pulp_exceptions.PulpCodedValidationException(
-                [PulpCodedException(error_codes.PLP1004, type=ConsumerGroup.collection_name,
-                                    object_id=group_id)])
+            raise pulp_exceptions.DuplicateResource(group_id), None, sys.exc_info()[2]
```

Comment:
@ipanova, sorry I was unclear. I think this is a good change, I meant that if you remove this, maybe we should remove this as well:
https://github.com/pulp/pulp/blob/25ae34a7a5db783eb3cac6d10e9080ba813fab6c/common/pulp/common/error_codes.py#L92